pom.xml

```diff
@@ -1625,10 +1625,13 @@
       </dependencyManagement>
     </profile>
     <profile>
-      <id>tez</id>
+      <id>tez0.9</id>
```

Comment:
If we only have one version Tez, we don't need the name tez + version.

Reply:
Ok, we corrected it.